community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -169,7 +174,7 @@
                         + ". There is a certain amount of minimum threads needed so for that reason there "
                         + "is no lower bound for this value. For optimal performance this value shouldn't be "
                         + "greater than the number of available processors." ),
-        STACKTRACE( "stacktrace", null,
+        STACKTRACE( "stacktrace", false,
                 "<true/false>",
                 "Enable printing of error stack traces." ),
         BAD_TOLERANCE( "bad-tolerance", 1000,
```

Comment:
At `CsvImporter.java:125` you change the default bad tolerance to 10 000, here it's 1000.